### PR TITLE
Support Composer 2

### LINKED ISSUE #218: upgrade composer api version
Composer just merge the branch 2.0 into master.

So we got this issue :
    - hirak/prestissimo 0.3.10 requires composer-plugin-api ^1.0.0 -> found composer-plugin-api[2.0.0] but it does not match your constraint.

### PR DESCRIPTION
**Pull request by kubawerlos**
Resolves https://github.com/hirak/prestissimo/issues/218

See https://github.com/hirak/prestissimo/issues/218#issuecomment-610581259

### CODE REVIEW

File: src/Plugin.php [hunk 1]
```diff
@@ -51,6 +51,11 @@ class Plugin implements
 
     public function activate(Composer $composer, IO\IOInterface $io)
     {
+        // Composer 2 comes with parallel downloads built-in
```
Comment (soullivaneuh) [inline src/Plugin.php:54]: So this plugin is kind of useless starting composer v2?

Do you have a link telling about such a feature? May be added to the comment.

**Review by soullivaneuh — CHANGES_REQUESTED** [review]

File: src/Plugin.php (hunk 1 above)
Comment (kubawerlos) [reply src/Plugin.php:54]: @soullivaneuh yes, it's in the description of the PR ☝️ - I'm not sure adding a link to the comment is a good idea, it is described in Composer's changelog: https://github.com/composer/composer/blob/master/CHANGELOG.md

Comment (podarok) [reply src/Plugin.php:54]: composer2 is slower than this plugin

Comment (kubawerlos) [reply src/Plugin.php:54]: @podarok then you have to contact @Seldaek and ask hin to remove parallel downloads built-in from Composer 2 ;)

**Comment by kubawerlos** [comment]
@hirak any chance for merge and release?

File: src/Plugin.php (hunk 1 above)
Comment (kubawerlos) [reply src/Plugin.php:54]: No, it's not: https://github.com/hirak/prestissimo/issues/233#issuecomment-716115604

**Comment by atapatel** [comment]
Waiting for support on composer 2.0.

**Comment by Seldaek** [comment]
This plugin is not needed anymore for Composer 2, so your best way forward is to remove your requirement on it as you upgrade to Composer 2. If this gets merged it'd make upgrading a bit smoother perhaps, but ultimately you are better off removing the plugin than installing it for nothing if you're using Composer 2.

**Comment by hirak** [comment]
I won't fix this plugin anymore, just use Composer 2.

**Comment by kubawerlos** [comment]
You could have merged this, so people upgrading from Composer 1 to 2 are not getting the incompatibility warning...

**Pull request closed without merging**